azkaban-web-server/src/main/java/azkaban/webapp/servlet/AbstractAzkabanServlet.java

```diff
@@ -58,6 +58,8 @@
   private static final String AZKABAN_FAILURE_MESSAGE =
       "azkaban.failure.message";
   private static final long serialVersionUID = -1;
+
+  protected String passwordPlaceholder;
```

Comment:
Why passwordPlaceholder is added in AbstractAzkabanServlet instead of LoginAbstractAzkabanServlet?